post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -54,13 +55,14 @@ public class PostProcessingService {
     private final ObjectMapper objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
 
     @Getter
-    private enum Entity {
+    enum Entity {
         ORGANIZATION(1, "organization", "organization_uid", "sp_nrt_organization_postprocessing"),
         PROVIDER(2, "provider", "provider_uid", "sp_nrt_provider_postprocessing"),
         PATIENT(3, "patient", "patient_uid", "sp_nrt_patient_postprocessing"),
         INVESTIGATION(4, "investigation", "public_health_case_uid", "sp_nrt_investigation_postprocessing"),
         NOTIFICATIONS(5, "notifications", "notification_uid", "sp_nrt_notification_postprocessing"),
-        F_PAGE_CASE(0, "investigation", "public_health_case_uid", "sp_f_page_case_postprocessing"),
+        LDF_DATA(6, "ldf_data", "ldf_uid", "sp_nrt_ldf_postprocessing"),
+        F_PAGE_CASE(0, "fact page case", "public_health_case_uid", "sp_f_page_case_postprocessing"),
```

Comment:
` F_PAGE_CASE` and `CASE_ANSWERS` have same enum numerical value?

Reply:
It's not a enum value, it's sorting order defining the order of processing. Entities with 0 (and negative) do not participate in sorting (they are processed inside of  Investigation case)